.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request. The three fields below are mandatory. -->
+
+## Summary
+
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
+
+## Changelog
+
+<!-- Help reviewers and the release process by writing your own changelog entry. For an example, see:
+https://reactnative.dev/contributing/changelogs-in-pull-requests -->
+
+[CATEGORY] [TYPE] - Message
+
+## Test Plan
+
+<!-- Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes the user interface. -->
+
+## Related issues/PRs
+
+<!-- If you haven't already, link to issues/PRs that are related to this change. This helps us develop the context and keep a rich repo history. If this PR is a continuation of a past PR's work, link to that PR. If the PR addresses part of the problem in a meta-issue, mention that issue. -->
```

Comment:
This field is so necessary! But, I think it would be good to be included in ```Summary``` field. - ```Which issue(s) this PR fixes:```